doc/api/async_hooks.md

```diff
@@ -859,6 +859,249 @@ for (let i = 0; i < 10; i++) {
 }
 ```
 
+## Class: AsyncLocalStorage
+<!-- YAML
+added: REPLACEME
+-->
```

Comment:
Is this going to be landed initially as experimental? If so, needs an indicator here.

Reply:
All of async_hooks is still experimental … ;)

Reply:
ah, right, carry on then ;-)